### DISCUSSION TITLE
Trying to change which-key back to defaults settings, and having issues

### DISCUSSION THREAD

**Discussion by bruteforks** [LunarVim/LunarVim#3697] [Q&A]
New to lunarvim, so apologize if this is a newb question.. but I've been trying to get the default functions back for which-key. The marks/registers one.

yet this isn't working:
`
lvim.builtin.which_key.setup.plugins.marks = true
`

**Comment by LostInTheLogs** [comment]
it works for me, are there any errors in `:messages`? also what's your :LvimVersion ?
